src/SFA.DAS.EmployerAccounts/Models/Account/AccountStats.cs

```diff
@@ -6,5 +6,7 @@ public class AccountStats
         public int PayeSchemeCount { get; set; }
         public int OrganisationCount { get; set; }
         public int TeamMemberCount { get; set; }
+        public int TeamMembersInvited { get; set; }
+        public int SignedAgreementCount { get; set; }   
```

Comment:
Don't think we need to pull SignedAgreementCount in stats

Reply:
Removed SignedAgreementCount from stats